src/containers/qtreetbl.c

```diff
@@ -882,7 +882,7 @@ bool qtreetbl_debug(qtreetbl_t *tbl, FILE *out) {
     }
 
     qtreetbl_lock(tbl);
-    print_node(tbl->root, out, NULL, true);
+    print_node(tbl->root, out, NULL, false);
```

Comment:
Not really a typo, doesn't matter for root.

Reply:
Yes, I figured it doesn't matter, that's why I changed it to 0 (false) value. Just a practice, parameters that are set to non-default value have to have a meaning. Otherwise, I'd think about it why? whenever I revisit the line. Great code!

Reply:
Yeah, makes sense.